app/controllers/observer_controller/indexes.rb

```diff
@@ -22,8 +22,10 @@ def observations_by_name
 
   # Displays matrix of Observations with the given text_name (or search_name).
   def observations_of_name
-    query = create_query(:Observation, :all, names: [params[:name]],
-                                             include_synonyms: true, by: :created_at)
+    query = create_query(:Observation,  :all,
```

Comment:
The changes in this file (to fix LineLength offenses) push the class size just over our RuboCop limit (250 lines).
I think we should deal with that later rather than holding up this PR.

Reply:
Agreed.